modules/test/playwright/fixtures/dataHelper.fixture.js

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
This was not approved unfortunately, we cannot use it. Please remove this commit